### PR TITLE
Add example scripts

### PR DESCRIPTION
**Pull request by Lars-DHondt-KUL**
## Description

Added example scripts to show how to use/combine certain settings
- ankle_exoskeleton
- batch_sensitivity_analysis

## Motivation and Context

Easier for new users to get started.

## How Has This Been Tested?

run examples

## Suggested tests for reviewers

run examples

### CODE REVIEW

**Comment by VDB-Bram** [comment]
Should there be information on these examples in the readme? Something like 'we provide several example scripts in /Examples ...'?

File: README.md [hunk 1]
```diff
@@ -53,6 +53,12 @@ The code is written such that as a user you only have to interact with [*main.m*
 
 All user-defined [settings](./Documentation/SettingsOverview.md) are stored in structure *S*. In main.m you have to specify the required settings and are free to change/add the optional settings. 
 
+The [examples folder](./Examples/) contains scripts illustrating how to use settings to:
+- Predict a gait pattern resulting from a limited number of muscle synergies.
+- Predict gait with assistance from an ankle exoskeleton.
+- Run a sensitivity analysis.
+
```
Comment (VDB-Bram) [inline README.md:60]: Wouldn't this fit better after the section 'Before running a simulation'?

File: Examples/ankle_exoskeleton.m [hunk 2]
```diff
@@ -0,0 +1,79 @@
+% --------------------------------------------------------------------------
+% ankle_exoskeleton
+%   Simulate walking with an ankle exoskeleton that provides a
+%   plantarflexion torque in function of the progression within a stride.
+%   Running this simulation requires the function desired_torque_generator,
+%   which can be downloaded from
+%   https://www.science.org/doi/suppl/10.1126/science.aal5054/suppl_file/aal5054_zhang_sm_data_s2.zip.
+%   Note that using the downloaded function will throw an error related to
+%   the variable prev_stride_time. To fix this, set it equal to
+%   stride_period.
```
Comment (VDB-Bram) [inline Examples/ankle_exoskeleton.m:10]: Maybe be more specific?

```suggestion
%   Note that using the downloaded function will throw an error related to
%   the variable prev_stride_time. To fix this, set `prev_stride_time = stride_time;`
%   in the beginning of the function desired_torque_actuator.
```

**Review by VDB-Bram — COMMENTED** [review]
Two examples ran without any issues.

Small remarks regarding documentation have been made.

File: README.md (hunk 1 above)
Comment (Lars-DHondt-KUL) [reply README.md:66]: Now it's under "how to use the code"

**Review by VDB-Bram — APPROVED** [review]

**Pull request merged**